### DISCUSSION TITLE
Palo Alto Firewall Logs Change to UTC

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#7746] [Unsupported Versions] [answered]
Good Evening All!

With the help of this discussion board I was able to ingest my Palo Alto firewall logs into SO using filebeat.  

However, my dilemma is that SO is interpreting the log time stamp as UTC, but is really in Los Angeles time zone. 

Is there a way to tell SO/Filebeat that the time is in Los Angeles time so it can covert it to UTC?

Thanks,

**Comment by dougburks** [comment, marked as answer]
It's usually best to configure the sender (in this case your Palo Alto firewall) to send logs in UTC avoiding conversion altogether.

If for some reason you can't do that and you are using the Palo Alto filebeat module, see:
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-panw.html#_time_zone_support_12

**Reply by oneCrazyAdmin** [reply]
Thanks for the direction.  It looks as though Palo Alto doesn't/can't send the timezone information in the syslog.  I am trying to use a processor and am having some trouble with the syntax.  I will post something when I have more information.

**Reply by oneCrazyAdmin** [reply]
I was able to add the processor as described, but I am guessing that I am missing something in the syntax because it is not be rendered in the elastic yml format

`filebeat:
    third_party_filebeat:
      modules:
         panw:
           panos:
              enabled: true
              var.syslog_host: 0.0.0.0
              var.syslog_port: 9002
              processors:
                 add_locale: ~`

and the salt stack generates it as 
`- module: panw
    panos:
       enabled: true
       var.syslog_host: 0.0.0.0
       var.syslog_port: 9002
       processors: {'add_locale': 'None'}`

am I missing something?

**Reply by dougburks** [reply]
> filebeat: third_party_filebeat: modules: panw: panos: enabled: true var.syslog_host: 0.0.0.0 var.syslog_port: 9002 processors: add_locale: ~

Looks like your filebeat config isn't indented properly.

> It looks as though Palo Alto doesn't/can't send the timezone information in the syslog

Instead of configuring the Palo Alto firewall to send the timezone information, another option might be to configure it to use UTC as its timezone internally.
